index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
           </pre>
         </section>
       </section>
+
+      <section>
+        <h3>Data Schemas</h3>
+
+        <p>
+Data schemas are useful when enforcing a specific structure on a given
+collection of data. There are at least two types of data schemas that this
+specification considers:
+        </p>
+
+        <ul>
+          <li>
+Data verification schemas, which are used to <a>verify</a> that the structure
+and contents of a <a>credential</a> or <a>verifiable credential</a> conform to a
+published schema.
+          </li>
+          <li>
+Data encoding schemas, which are used to map the contents of a
+<a>verifiable credential</a> to an alternative representation format, such as a
+format used in a zero-knowledge proof.
+          </li>
+        </ul>
+
+        <p>
+It is important to understand that data schemas serve a different purpose from
+the <code>@context</code> property, which neither enforces data structure or
+data syntax, nor enables the definition of arbitrary encodings to alternate
+representation formats.
```

Comment:
```suggestion
the <code>@context</code> property.
```

Reply:
I'd avoid talking about what `@context` does here... Possibly reference another section with positive language.

Reply:
See comments in https://github.com/w3c/vc-data-model/pull/1203/files#r1279865135